### PR TITLE
Table Contents are now Visible in Dark Mode

### PR DESCRIPTION
**Pull request by anushri20**
Table Contents are now Visible in Dark Mode

Fixes # 483

#### Changes done:
- [x] Task 1 changes done custom.scss file

#### Screenshots

image

image

#### ✅️ By submitting this PR, I have verified the following

- [ x] Checked to see if a similar PR has already been opened 🤔️
- [ x] Reviewed the contributing guidelines 🔍️
- [x ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x ] Tried Squashing the commits into one

### CODE REVIEW

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
Could we fix the hound issues.

**Comment by anushri20** [comment]
@YashKumarVerma  Yeah, I can try.

File: _sass/custom/custom.scss [hunk 1]
```diff
@@ -277,4 +277,7 @@ $text-color: #111111;
 	}
     }
 }
+.table-wrapper {
+  color: $black;
```
Comment (ayushontop) [inline _sass/custom/custom.scss:281]: I think green color will look more cool. what you think?

Comment (anushri20) [reply _sass/custom/custom.scss:281]: Since the background is white i am not very sure about the green colour.

**Comment by manjotsidhu** [comment]
Can be closed, better implementation done in #528

**Comment by manjotsidhu** [comment]
> Can be closed, better implementation done in #528

@Shivansh2407

**Comment by Shivansh2407** [comment]
> Can be closed, better implementation done in #528

**Pull request closed without merging**